packages/guideline-blocks-settings/src/components/Attachments/AttachmentItem.tsx

```diff
@@ -115,7 +116,8 @@ export const AttachmentItem = forwardRef<HTMLButtonElement, AttachmentItemProps>
                                 isDragging || isOverlay
                                     ? 'tw-cursor-grabbing tw-bg-button-background-pressed hover:tw-bg-button-background-pressed'
                                     : 'tw-cursor-grab hover:tw-bg-button-background-hover',
-                                isFocusVisible && FOCUS_STYLE,
+                                isFocusVisible &&
+                                    'tw-ring-4 tw-ring-blue tw-ring-offset-2 dark:tw-ring-offset-black tw-outline-none',
```

Comment:
We can probably use this util class from Fondue:
```suggestion
                                isFocusVisible &&
                                    'tw-ring-focus,
```